src/apigility-ui/service/documentation.service.js

```diff
@@ -0,0 +1,88 @@
+(function () {
+  'use strict';
+
+  angular
+    .module('apigility.service')
+    .service('documentation', DocumentationService);
+
+  DocumentationService.$inject = [ ];
+
+  function DocumentationService() {
+
+    function hasHalMediaType (mediatypes) {
+      if (typeof mediatypes !== 'object' || !Array.isArray(mediatypes)) {
```

Comment:
Considering you're calling `.forEach` on the result, I think this needs to be just the `Array.isArray` check. I can make that change during merge.